package/libs/openssl/Makefile

```diff
@@ -189,17 +328,17 @@ define Build/Configure
 			--prefix=/usr \
 			--openssldir=/etc/ssl \
 			$(TARGET_CPPFLAGS) \
-			$(TARGET_LDFLAGS) -ldl \
-			$(if $(CONFIG_OPENSSL_OPTIMIZE_SPEED),,-DOPENSSL_SMALL_FOOTPRINT) \
-			$(OPENSSL_NO_CIPHERS) \
+			$(TARGET_LDFLAGS) $(if $(CONFIG_OPENSSL_USE_VERSION_1_0_2),,-ldl) \
```

Comment:
`-ldl` looks weird here. I built a seperate 1.1.1 package successfully without it. This might be fixed, but I'm not sure.

Reply:
In fact, I think I have it backwards.  The original line, for 1.0.2 had the `-ldl`, and then when I changed to 1.1.0, I dropped, and meant to restore it for 1.0.2.  However, after the change, I built 1.0.2 for every `CONFIG_ARCH` availabe, and none of them failed, so I'll drop it for all versions.  Thanks for pointing it out.

Reply:
This might be leftover from an earlier version.  The added openwrt configurations in `Configure` all set `$lflags` to `-ldl`, so this is definitely not needed.